src/oc_erchef/habitat/config/elasticsearch-index-init.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Can we move this to a location inside the erchef build so we can easily access it from the old code and the new code?